core/zms/src/main/rdl/Authority.rdli

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 Athenz Authors
+// Licensed under the terms of the Apache version 2.0 license. See LICENSE file for terms.
+
+//List of valid user authority attribute values
+type UserAuthorityValidValuesList Struct {
```

Comment:
Does it make sense to combine the attributes into a single call and just return the type as part of the response?

Something like:

type UserAuthorityAttributeValue Struct {
    SimpleName name;
    SimpleName type;
}

UserAuthorityAttributeValueList Struct {
    Array<UserAuthorityAttributeValue> values;
}

Reply:
Sure but because each type may have several attributes I think I'll hold it as a map (type->List of attributes). Let me know what you think